src/commonMain/kotlin/com/algolia/search/serialize/Internal.kt

```diff
@@ -33,7 +33,7 @@ internal val regexSearchable = Regex("^$KeySearchable\\((.*)\\)$")
 internal val regexFacet = Regex("^\\{facet:(.*)\\}$")
 internal val regexPlaceholder = Regex("^<(.*)>$")
 internal val regexPoint = Regex("^(.*),(.*)$")
-internal val regexUserToken = Regex("^[a-zA-Z0-9_-]*$")
+internal val regexUserToken = Regex("^[a-zA-Z0-9_\\-\\.\\:]*\$")
```

Comment:
Nitpicky: If you want to put a comment as to why we're adding this (for ipv4 and ipv6)

Reply:
Yup! Good idea 👍